confgenerator/config.go

```diff
@@ -153,6 +169,11 @@ func newValidator() *validator.Validate {
 		}
 		return t >= tmin
 	})
+	// filter validates that a Cloud Logging filter condition is valid
+	v.RegisterValidation("filter_list", func(fl validator.FieldLevel) bool {
```

Comment:
I think you can simplify this by getting rid of `newFilters`, making the validator check a single string, and then using `required,dive,filter` as the validation rule to check each element separately.

Reply:
Thanks! TIL about `dive`, that's much better. For an added bonus the error message also tells you the index of the element that fails validation which we didn't have before.